exercises/grains/grains_test.rb

```diff
@@ -1,45 +1,81 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'grains'
 
+# Test data version: aa12f2e
 class GrainsTest < Minitest::Test
-  def test_square_1
+  def test_1
+    # skip
     assert_equal 1, Grains.square(1)
   end
 
-  def test_square_2
+  def test_2
     skip
     assert_equal 2, Grains.square(2)
   end
 
-  def test_square_3
+  def test_3
     skip
     assert_equal 4, Grains.square(3)
   end
 
-  def test_square_4
+  def test_4
     skip
     assert_equal 8, Grains.square(4)
   end
 
-  def test_square_16
+  def test_16
     skip
-    assert_equal 32_768, Grains.square(16)
+    assert_equal 32768, Grains.square(16)
   end
 
-  def test_square_32
+  def test_32
     skip
-    assert_equal 2_147_483_648, Grains.square(32)
+    assert_equal 2147483648, Grains.square(32)
   end
 
-  def test_square_64
+  def test_64
     skip
-    assert_equal 9_223_372_036_854_775_808, Grains.square(64)
+    assert_equal 9223372036854775808, Grains.square(64)
```

Comment:
When we get to numbers like this the underscore representation is REALLY helpful.